### PR TITLE
Update SuiteSparse to 7.5.1 with embedded Metis

### PR DESCRIPTION
**Pull request by NeroBurner**
The following big changes have been done:

  - fixes Metis based segmentation fault [#123](https://github.com/jlblancoc/suitesparse-metis-for-windows/issues/123)
  - remove `SuiteSparse::metis` target as Metis is now a private dependency of `CHOLMOD` module
  - remove option `METIS_DIR`
  - always compile `SuiteSparse_config` with `-DNTIMER` disabling tic-toc support
  - compile `CXSparse` without complex support
  - remove SourceWrappers for `CHOLMOD` and `CXSparse` modules

The changes are split up into fitting commits with more information on the specific change.

Should we just remove the new not (by us) supported SuiteSparse modules to save space?

### CODE REVIEW

**Comment by NeroBurner** [comment]
squashed the log commits to have one clean commit that changes the changelog entry

File: CHANGELOG.md
Comment (jlblancoc) [inline CHANGELOG.md:1]: Great work!

I just can't check all upstream changes, but if CI is happy and you checked it, I'm more than happy!

On deleting the new modules... Once the commit is there, it won't save disk/repo space unless force pushes are used, so I tend to think it's better to just live with them for now.

**Review by jlblancoc — APPROVED** [review]
Great work!

I just can't check all upstream changes, but if CI is happy and you checked it, I'm more than happy!

On deleting the new modules... Once the commit is there, it won't save disk/repo space unless force pushes are used, so I tend to think it's better to just live with them for now.

**Comment by NeroBurner** [comment]
released: https://github.com/jlblancoc/suitesparse-metis-for-windows/releases/tag/v7.5.1-1

**Pull request merged**